pithos/plugins/notify.py

```diff
@@ -77,3 +80,47 @@ def on_disable(self):
         if self._shutdown_handler:
             self._app.disconnect(self._shutdown_handler)
             self._shutdown_handler = 0
+
+
+class NotifyPluginPrefsDialog(Gtk.Dialog):
+    __gtype_name__ = 'NotifyPluginPrefsDialog'
+
+    def __init__(self, window, settings):
+        super().__init__(use_header_bar=1)
+        self.set_title(_('Notification Preferences'))
+        self.set_default_size(300, -1)
+        self.set_resizable(False)
+        self.connect('delete-event', self.on_close)
+
+        self.pithos = window
+        self.settings = settings
+        self.show_album = literal_eval(self.settings['data']) if self.settings['data'] else False
+
+        box = Gtk.Box()
+        label = Gtk.Label()
+        label.set_markup('<b>{}</b>\n{}'.format(_('Show Album Info'), None))
```

Comment:
```suggestion
        label.set_markup('<b>{}</b>\n'.format(_('Show Album Info')))
```

Surely you meant this.

Reply:
It was weirdly generating errors previously because it needed a second parameter. That seems to have resolved so I've removed the second parameter and the new-line while adding a valign setting to the label as well.